### PR TITLE
Secondary branch

### PR DESCRIPTION
**Pull request by Jrmcuervo**
In this pull requests I create a hello microverse project and set it up, also I create a readme and a ignore file.

### CODE REVIEW

File: hello-microverse.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: green;
+}
```
Comment (Owusu-Desmond) [inline hello-microverse.css]: - [ ] Please from the project requirement, your CSS file should include one class with styles for your header (make it in your favorite color). 
- [ ] Also, I can see this file is not linked to your HTML file.

File: hello-microverse.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<html>
+  <body>
+    <h1>Hello Microverse!</h1>
+  </body>
```
Comment (Owusu-Desmond) [inline hello-microverse.html]: - [ ] I suggest you should give your `<h1>Hello Microverse!</h1>` a class attribute like `<h1 class="header">Hello Microverse!</h1>` and use the `header` class to style it.
- [ ] Also, I can see this file is not linked to your CSS file.

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Jrmcuervo , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [ ] From the project requirements, Your `test.md` file with the text "This file should be ignored by git" and make sure that it is actually ignored by git and is not present in your GitHub repository. But as you can see you still have it in this GitHub repository.

## Optional suggestions

- [ ] It will be better if you rename your `hello-microverse.html` to `index.html`

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: hello-microverse.css (hunk 1 above)
Comment (Jrmcuervo) [reply hello-microverse.css]: Hello Desmond, I think now it's done, please let me know if I have to make more changes, Thanks a lot.

File: hello-microverse.html (hunk 2 above)
Comment (Jrmcuervo) [reply hello-microverse.html]: These changes are also modified as you ask, please tell me if it is anything more to do, again thanks a lot.

**Comment by Jrmcuervo** [comment]
All requested changes were modified on request, please let me know if there
is anything else I can do, regards.
Have an excellent day!
-Rubén

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Jrmcuervo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**